tenets/codelingo/go/global-var/codelingo.yaml

```diff
@@ -0,0 +1,18 @@
+tenets:
+  - name: global-variable
+    doc: |
+      Check that no globals are present in Go code.
+    flows:
+      codelingo/review:
+        comment: Global variables are an input to functions that is not visible in the functions signature, complicate testing, reduces readability and increase the complexity of code.
+    query: |
+      import codelingo/ast/go
+      
+      go.file(depth = any):
+        go.decls:
+          go.value_spec(depth = 1):
+            go.names:
+              @playground.highlight
+              go.ident:
+                private == "true"
+                public == "false"
```

Comment:
These two lines say the same thing, you can remove one.

Reply:
In fact, why not match public globals too?

Reply:
Public globals may be used by another package

Reply:
Is this a reason for or against allowing public globals?